packages/caver-klay/caver-klay-accounts/src/index.js

```diff
@@ -875,6 +892,68 @@ Accounts.prototype.signTransactionWithSignature = function signTransactionWithSi
     });
 };
 
+/**
+ * combineSignatures combines RLP encoded raw transaction strings.
```

Comment:
It should be more descriptive.
For example, it seems that this function can also receives transactions even if 'feePayer' field is different.
What other assumptions does this function have?

Reply:
In combineSignatures function, transaction should be same. 
But after sender sign to transaction, there is not fee payer's address in rawTransaction string(fee payer will be '0x' after decode). 
So for handling this, when check transaction, if fee payer is '0x', then don't compare.

Reply:
Okay. Please add that condition to the comment of this function.

Reply:
I added descriptions and comments :)